lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -1894,6 +1894,36 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
         rewriter.replaceOpWithNewOp<Torch::AtenOneHotOp>(binder.op, resultType,
                                                          argmax, oneInt);
 
+        return success();
+      });
+  patterns.onOp(
+      "LpNormalization", 22,
```

Comment:
I think we can lower since version to 1 because the only difference between 1 and 22 is that 22 supports bfloat16

https://onnx.ai/onnx/operators/text_diff_LpNormalization_1_22.html